### PR TITLE
increasing worker memory for s390x upgrade jobs

### PR DESCRIPTION
**Pull request by LakshmiRavichandran1**

### CODE REVIEW

File: ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh [hunk 1]
```diff
@@ -129,6 +129,8 @@ done
 yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.template.spec.providerSpec.value[domainMemory] ${WORKER_MEMORY}
 # Bump the libvirt workers disk to to 30GB
 yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.template.spec.providerSpec.value.volume[volumeSize] ${WORKER_DISK}
+#  Bump the vcpu to 8
+yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.template.spec.providerSpec.value.volume[domainVcpu] 8
```
Comment (jaypoulz) [inline ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh:133]: Can we parameterize this as well.

I'm fine with 8 as the default.

Comment (LakshmiRavichandran1) [reply ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh]: addressed in the recent commit.

**Pull request closed without merging**